packages/edit-widgets/src/index.js

```diff
@@ -24,6 +24,16 @@ import './store';
 import './filters';
 import * as widgetArea from './blocks/widget-area';
 import Layout from './components/layout';
+import {
+	ALLOW_REUSABLE_BLOCKS,
+	ENABLE_EXPERIMENTAL_FSE_BLOCKS,
+} from './constants';
+
+const disabledBlocks = [
+	'core/more',
+	'core/freeform',
+	...( ! ALLOW_REUSABLE_BLOCKS && [ 'core/block' ] ),
```

Comment:
@getdave this does not work if we set ALLOW_REUSABLE_BLOCKS to true as the expression will return `false` which is not iterable.

Reply:
Good spot. Fix available for review https://github.com/WordPress/gutenberg/pull/34634